index.js

```diff
@@ -1,9 +1,33 @@
 'use strict';
-const urlRegex = require('url-regex');
+const urlRegex = require('url-regex')();
 const normalizeUrl = require('normalize-url');
+const queryString = require('query-string');
```

Comment:
You can just use the built-in `require('url').URLSearchParams`

Reply:
done